chsdi/models/vector/uvek.py

```diff
@@ -393,6 +393,88 @@ class unf_pers_velo(Base, Vector):
 register('ch.astra.unfaelle-personenschaeden_fahrraeder', unf_pers_velo)
 
 
+class schwerverunf_kanton_alkohol(Base, Vector):
```

Comment:
Minor but for class name sthe convention is as follow:

1. The first letter is a captial letter
2. We use camelCase